stable/yugabyte/templates/service.yaml

```diff
@@ -386,7 +386,7 @@ spec:
               --undefok=num_cpus,enable_ysql \
               --use_node_hostname_for_local_tserver=true \
             {{- if $root.Values.authCredentials.ysql.password }}
-              --ysql_enable_auth=true \
+              --ysql_enable_auth=true 
```

Comment:
This looks like a typo. This line should be removed.

Reply:
@tvesely : is Stanley around to update this PR? Otherwise, can you please send a new PR with the typo removed?